PyPDF2/_encryption.py

```diff
@@ -611,17 +625,21 @@ def __init__(
     ) -> None:
         # See TABLE 3.18 Entries common to all encryption dictionaries
         self.algV = algV
+        self.algR = algR
         self.entry = entry
         self.key_size = entry.get("/Length", 40)
         self.id1_entry = first_id_entry
         self.StmF = StmF
         self.StrF = StrF
         self.EFF = EFF
 
+        # 1 => owner password
+        # 2 => user password
+        self._password_type = 0
```

Comment:
I think an Enum might be easier:

```suggestion
from enum import Enum

class PasswordType(Enum):
    NOT_DECRYPTED = 0
    OWNER_PASSWORD = 1
    USER_PASSWORD = 2
    
 ....
 
     self._password_type = 0


        # 1 => owner password
        # 2 => user password
        self._password_type = PasswordType.NOT_DECRYPTED
```